examples/jsm/renderers/common/Backend.js

```diff
@@ -90,6 +90,8 @@ class Backend {
 
 	// utils
 
+	resolveTimeStampAsync( renderContext, type ) { }
```

Comment:
Shouldn't it be `Timestamp` instead of `TimeStamp` for consistency? /ping @RenaudRohlinger

Reply:
Yep! 👍